runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -392,7 +392,7 @@ void LightningSimulator::PartialCounts(DataView<double, 1> &eigvals, DataView<in
     }
 }
 
-auto LightningSimulator::Measure(QubitIdType wire) -> Result
+auto LightningSimulator::Measure(QubitIdType wire, int8_t postselect) -> Result
```

Comment:
You also need to update the signature in `LightningKokkosSimulator.cpp` and `OpenQasmDevice.cpp`. But, don't worry about implementing this support for those devices. You can add `RT_FAIL_IF` to raise runtime errors if users try to use the postselect on these devices.

Reply:
Done